tools/yarn_install.R

```diff
@@ -46,13 +46,34 @@ needs_prefix <- c(
   "backface-visibility", "touch-action"
 )
 for (prop in needs_prefix) {
-  scss_src <- lapply(scss_src, add_prefixes, prop)
+  scss_src <- lapply(scss_src, add_property_prefixes, prop)
 }
 # Conditionally prefix text-decoration if its not CSS2 compliant
 # https://www.w3.org/TR/CSS2/text.html#lining-striking-props
 # https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration
 # https://caniuse.com/#search=text-decoration
-scss_src <- lapply(scss_src, add_prefixes, "text-decoration", c("none", "underline", "overline", "line-through", "blink", "inherit"))
+scss_src <- lapply(
+  scss_src, add_property_prefixes, "text-decoration",
+  c("none", "underline", "overline", "line-through", "blink", "inherit")
+)
+
+
+add_value_prefixes <- function(src, value) {
+  pattern <- paste0("^\\s*(.+):\\s*", value)
+  idx <- grep(pattern, src)
+  for (i in idx) {
+    prop_val <- strsplit(src[[i]], ":\\s*")[[1]]
+    src[[i]] <- paste0(
+      prop_val[1], ": ", c("-webkit-", "-moz-", "-ms-"), prop_val[2],
```

Comment:
In `add_property_prefixes()`, `prop` should come after the prefixed css rules.  This could be taken care of when updating the prefix values, also removing the need for pasting the old value with the new values.


(in `add_property_prefixes()`) Change to:
```
    src[[i]] <- prop_prefixes
```